uniffi_bindgen/src/bindings/python/gen_python/mod.rs

```diff
@@ -378,7 +378,7 @@ pub mod filters {
     }
 
     pub fn ffi_converter_name(as_ct: &impl AsCodeType) -> Result<String, askama::Error> {
-        Ok(as_ct.as_codetype().ffi_converter_name())
+        Ok(String::from("_Uniffi") + &as_ct.as_codetype().ffi_converter_name()[3..])
```

Comment:
IIUC this is transforming, say, `FfiConverterFoo` to `_UniffiConverterFoo`, right? In some ways I kinda prefer `UniffiFfiConverter` - it's mildly redundant, but it means the fact it's an `ffi converter` is more obvious, and best I can tell it probably doesn't hurt, right?

Alternatively, it's not particularly important for everything to have a `_Uniffi` prefix is it? eg, `_rust_call` functions below didn't get one - they just got a leading `_` - so is there any reason `_FfiConverterFoo` isn't just as good for the problem being solved here? eg, I had to go down a strange path to work out why the `3..` existed here, whereas just tacking on a leading `_` seems far less likely to have people scratching their head.

Reply:
> eg, I had to go down a strange path to work out why the `3..` existed here, whereas just tacking on a leading `_` seems far less likely to have people scratching their head.

Ah yea, that's a hack. I guess I should fix that before merging…

> eg, `_rust_call` functions below didn't get one - they just got a leading `_`

Ah, I probably just missed those… Having everything get a `uniffi` prefix makes it quite clear that nothing will conflict and also sets a standard for future symbols, I'd call that a win. I guess I'll add `uniffi` to those symbols as well.